_tests/integration/ios_test.go

```diff
@@ -694,26 +700,43 @@ configs:
       - pull_request_source_branch: '*'
         workflow: primary
       workflows:
-        primary:
+        deploy:
+          description: |
+            Deploys app using *Deploy to bitrise.io* step.
+
+            Next steps:
+            - Set up [Connecting to an Apple service with API key](https://devcenter.bitrise.io/en/accounts/connecting-to-services/connecting-to-an-apple-service-with-api-key.html##).
+            - Or further customise code signing following our [iOS code signing](https://devcenter.bitrise.io/en/code-signing/ios-code-signing.html) guide.
           steps:
           - activate-ssh-key@%s:
               run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
           - git-clone@%s: {}
           - cache-pull@%s: {}
-          - script@%s:
-              title: Do anything with Script step
-          - certificate-and-profile-installer@%s: {}
           - xcode-archive@%s:
               inputs:
-              - project_path: $BITRISE_PROJECT_PATH
-              - scheme: $BITRISE_SCHEME
-              - distribution_method: $BITRISE_DISTRIBUTION_METHOD
```

Comment:
Discussed that *distribution_method*  has `development` not `$BITRISE_DISTRIBUTION_METHOD` as default value, needs to be included.

Reply:
Thanks for pointing it out. Fixed it.